src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataStreamTest/DataStreamTest.cs

```diff
@@ -2009,11 +2009,12 @@ private static void TimeoutDuringReadAsyncWithClosedReaderTest(string connection
                 }
                 proxy.Stop();
             }
-            catch (SqlException ex)
+            catch (SqlException)
             {
                 // In case of error, stop the proxy and dump its logs (hopefully this will help with debugging
+                // re-throwing exception will change the stack. we just throw the caught exception.
```

Comment:
Since this is the recommended rule, comments like these are unnecessary imo.

Reply:
done.